### PR TITLE
refactor: update module

### PR DESCRIPTION
**Pull request by ricardogobbosouza**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,16 +7,15 @@
...
-    "types"
+    "dist"
   ],
-  "main": "lib/index.js",
-  "types": "types/index.d.ts",
   "scripts": {
     "build": "tsc",
-    "lint": "eslint --ext .js,.vue .",
-    "release": "yarn test && standard-version && git push --follow-tags && npm publish",
+    "lint": "eslint --ext .ts .",
```
Comment (pi0) [inline package.json:17]: We would also need `.js,.vue` for fixtures

File: package.json [hunk 2]
```diff
@@ -7,16 +7,15 @@
...
+    "release": "yarn test && yarn build && standard-version && git push --follow-tags && npm publish",
```
Comment (pi0) [inline package.json:18]: Neat: Probably not needed for us but doing `build` after version, ensures if bundling version, we use updated one

File: src/nuxt.ts [hunk 3]
```diff
@@ -31,8 +30,9 @@ export async function loadFixture() {
   }
 }
 
-export async function loadNuxtPackage(name: 'nuxt') {
+export async function loadNuxtPackage () {
```
Comment (pi0) [inline src/nuxt.ts:33]: It is a general utility with type hint :) But change is ok :+1:

File: package.json (hunk 1 above)
Comment (ricardogobbosouza) [reply package.json]: Fixed

**Pull request merged**